shared/chat/conversation/messages/failure.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import React from 'react'
+import {globalColors, globalStyles} from '../../../styles'
+import {Text} from '../../../common-adapters'
+
+const Failure = ({failureDescription, onShowEditor, onRetry}: {failureDescription?: ?string, onRetry: () => void, onShowEditor: (event: any) => void}) => {
+  const error = `Failed to send${failureDescription ? ` -  ${failureDescription}` : ''}. `
+  const resolveByEdit = failureDescription === 'message is too long'
+  return (
+    <Text type='BodySmall'>
```

Comment:
This is to make it cross platform, cause we don't use `View` adapter anymore?

This was originally in retry.js (not written by me).